### PR TITLE
85 verification messages

### LINKED ISSUE #85: Add messages for additional verification in Concent
Add new messages MessageSubtaskResultsVerify and MessageAckSubtaskResultsVerify and MessageSubtaskResultsVerdict that are needed by Concent to enable additional verification. No logic in golem is needed now, just messages.
See [documentation](https://docs.google.com/document/d/1QMnamlNnKxichfPZvBDIcFm1q0uJHMHJPkCt24KElxc/edit#heading=h.dhlbf3h55z2u) and [documentation](https://docs.google.com/document/d/1QMnamlNnKxichfPZvBDIcFm1q0uJHMHJPkCt24KElxc/edit#heading=h.z08tf29b855e).

### PR DESCRIPTION
**Pull request by shadeofblue**
Add Messages:

+ `SubtaskResultVerify`
+ `AckSubtaskResultVerify`
+ `SubtaskResultSettled`

+ use `factory_boy` to generate the automate generation of test objects within tests
+ tests

closes #85

### CODE REVIEW

**Comment by jiivan** [comment]
@shadeofblue please mark your PR according to https://help.github.com/articles/closing-issues-using-keywords/

To fix test you may want to update `setup.py -> install_requires`

**Comment by shadeofblue** [comment]
@jiivan okay, updated

**Comment by shadeofblue** [comment]
@jiivan cleared/silenced pylint issues

File: golem_messages/message/tasks.py [hunk 1]
```diff
@@ -157,13 +157,18 @@ class SubtaskResultAccepted(base.Message):
         'payment_ts'
     ] + base.Message.__slots__
 
+#
+# @todo for consistency's sake, the name of this message class should be:
+#       `SubtaskResultsRejected` (+ that's what's specified in the docs)
+#
```
Comment (jiivan) [inline golem_messages/message/tasks.py]: Can you make an issue for this TODO, and put a link here?

**Pull request merged**